CHANGELOG.md

```diff
@@ -16,6 +16,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [Change Skin](https://wiki.vg/Mojang_API#Change_Skin) endpoint.
 - [Reset Skin](https://wiki.vg/Mojang_API#Reset_Skin) endpoint.
 - [Blocked Servers](https://wiki.vg/Mojang_API#Blocked_Servers) endpoint.
+- [Authenticate](https://wiki.vg/Authentication#Authenticate) endpoint.
+- [Refresh](https://wiki.vg/Authentication#Refresh) endpoint.
+- [Validate](https://wiki.vg/Authentication#Validate) endpoint.
```

Comment:
Validate method already exists in the library. Please, replace this line with a note about fixed endpoint URL.